src/lib/clients/node-client.ts

```diff
@@ -1,9 +1,33 @@
 import { Client } from './client'
-import { Config } from '../../types'
+import { Config, TokenSearchRules, TokenOptions } from '../../types'
+import { Token } from '../token'
 
 class MeiliSearch extends Client {
+  tokens: Token
+
   constructor(config: Config) {
     super(config)
+    this.tokens = new Token(config)
+  }
+
+  /**
+   * Generate a tenant token
+   * @memberof MeiliSearch
+   * @method generateTenantToken
+   * @param {string} dumpUid Dump UID
+   * @returns {String} Token
+   */
+  generateTenantToken(
+    searchRules: TokenSearchRules,
+    options?: TokenOptions
+  ): string {
+    if (typeof window === 'undefined') {
```

Comment:
The `esm` build uses the node client. But the `esm` build can be used in both a node and browser environment when the bundler uses the `module` field of the `package.json`. 
Because of that, I know this is a bit counterintuitive, the check still has to be done